src/messagetypes/message.py

```diff
@@ -1,24 +1,30 @@
+"""
+src/messagetypes/message.py
+===========================
+"""
 from debug import logger
 from messagetypes import MsgBase
 
 
 class Message(MsgBase):
-    def __init__(self):
-        return
+    """Encapsulate a message"""
+    # pylint: disable=attribute-defined-outside-init
 
     def decode(self, data):
+        """Decode a message"""
         # UTF-8 and variable type validator
-        if type(data["subject"]) is str:
+        if isinstance(data["subject"], str):
             self.subject = unicode(data["subject"], 'utf-8', 'replace')
         else:
             self.subject = unicode(str(data["subject"]), 'utf-8', 'replace')
-        if type(data["body"]) is str:
+        if isinstance(data["subject"], str):
             self.body = unicode(data["body"], 'utf-8', 'replace')
         else:
             self.body = unicode(str(data["body"]), 'utf-8', 'replace')
 
     def encode(self, data):
-        super(Message, self).encode()
+        """Encode a message"""
+        super(Message, self).__init__()
```

Comment:
is this even necessary?

Reply:
I removed encode and use __init__ instide of that. Because pylint shows so many errors like, old-style-class, variable should be defined under init, no-init called. So, i think that the best alternative to that.